user-guide/Troubleshooting/Known_issues/KI_ButtonState_shapes_in_Visual_Overview_fail_to_hide_as_expected.md

```diff
@@ -0,0 +1,35 @@
+---
+uid: KI_ButtonState_shapes_in_Visual_Overview_fail_to_hide_as_expected
+---
+
+# ButtonState shapes in Visual Overview fail to hide as expected
+
+## Affected versions
+
+- DataMiner 10.4.8, if your Cube client does not have internet access.
+
+- Cube clients that have been enforced to use Cube version 10.4.2425.2536 or lower, which can be configured using the *Force a specific version* setting in *System Center* > *System settings* > *Manage client versions*.
```

Comment:
10.4.2425.2536 is the version in which the issue is fixed, so it are the versions lower than this one, will be 10.4.2425.2072 in most cases

Reply:
```suggestion
- Cube clients that have been enforced to use a Cube version prior to 10.4.2425.2536, which can be configured using the *Force a specific version* setting in *System Center* > *System settings* > *Manage client versions*.
```